graphql/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
```

Comment:
Maybe let's leave it out till we figure out how we want to pull in the existing graphql-parser? From my limited experimenting with it, it was easiest to treat it as an external crate even if we fork it into this repo

Reply:
@queerviolet yep that sounds good to me. I mainly wanted to show that workspaces / multi crates was easy to setup. Happy to remove!

Reply:
It is good to see, I hadn't known about this feature, actually!